package.json

```diff
@@ -13,7 +13,7 @@
     "node": ">=0.12.0"
   },
   "scripts": {
-    "test": "xo && npm run prepublish && nyc ava",
+    "test": "xo && npm run prepublish && nyc ava --serial",
```

Comment:
Maybe, or something with a newer AVA release. Does the tests pass if you set `"screenshot-stream": "3.0.0"` and reinstall dependencies and rerun the tests? I'm on slow 3G today, so can't test.

I don't think making it `--serial` is the correct solution, though.